documentation/BrowserChannel.md

```diff
@@ -24,39 +24,45 @@ Also, this component create notifications for every connected client at any time
 This small and documented client illustrates the workflow.
 
 ```javascript
-var io = require("socket.io-client");
+var io = require("ws");
```

Comment:
replace "io" with "ws", otherwise this "small and documented client" will fail with a "ws is not defined" error.